packages/fast-development-site-react/src/components/breadcrumb/breadcrumb-item.tsx

```diff
@@ -8,11 +8,11 @@ export interface IBreadcrumbItemProps {
 }
 
 export interface IBreadcrumbItemManagedClasses {
-    breadcrumb__list__item: string;
+    breadcrumb_list__item: string;
```

Comment:
I think you actually want to camel-case `listItem` - the list-item is the thing that is a child of the breadcrumb